chapters/ch10/2.judgments-and-logging.ipynb

```diff
@@ -145,29 +145,27 @@
     "\n",
     "Create a feature set, the first feature retrieves the relevance score of the search string in the `title` field (hence `title:(${keywords})`), the second feature the same for `overview`, finally the third feature is simply the `release_year` of the movie. \n",
     "\n",
-    "We create a feature store named `movie_model` in Solr. We'll use the feature store name as a handle when we want to log features farther down."
+    "We create a feature store named `movie_model` in out search engine. We'll use the feature store name as a handle when we want to log features farther down."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 21,
+   "execution_count": 25,
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/plain": [
        "[{'name': 'title_bm25',\n",
-       "  'class': 'org.apache.solr.ltr.feature.SolrFeature',\n",
-       "  'params': {'q': 'title:(${keywords})'},\n",
-       "  'store': 'movie_model'},\n",
+       "  'template': {'match': {'title': '{{keywords}}'}},\n",
```

Comment:
This looks like OpenSearch syntax, but there's Solr syntax in later cells in this notebook? Was part of this notebook run with Solr and part with OpenSearch? Assuming so, the default that needs to be committed is the Solr version.

Reply:
I've rerun these.